File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	
	
	List<TbBrand> findAll();
	
	PageResult findByPage(TbBrand brand, int pageNum, int pageSize);
	
	void save(TbBrand tbBrand) throws Exception;

	TbBrand findById(Long id) ;

	void update(TbBrand brand) throws Exception;

	void delete(Long [] ids) throws Exception;

	List<Map> selectOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService', function($http) {
		
		this.findAll = function() {
			return $http.get('../brand/findAll.do');
		}

		this.findPage = function(page, size) {
			return $http.get('../brand/findPage.do?page=' + page + '&pageSize='+ size);
		}

		this.add = function(entity) {
			return $http.post('../brand/save.do', entity);
		}

		this.update = function(entity) {
			return $http.post('../brand/update.do', entity);
		}
		
		this.findOne = function(id){
			return $http.get('../brand/findById.do?id=' + id);
		}
		
		this.del = function(ids){
			return $http.post('../brand/del.do', ids);
		}
		
		this.search = function(page,size,searchEntity){
			return $http.post('../brand/search.do?page=' + page + '&pageSize=' + size,searchEntity);
		}
		
		this.selectOptionList = function(){
			return $http.get('../brand/selectOptionList.do');
		}
	});